waiter/config-full.edn

```diff
@@ -752,6 +752,9 @@
  ; ---------- Load Balancing ----------
 
  :work-stealing {
+                 ;; The maximum number of outstanding work-stealing offers per service per router:
+                 :max-work-stealing-offers-per-service 50
```

Comment:
```suggestion
                 :max-work-stealing-offers-per-service 20
```

Reply:
I think it's good to keep the `config-full.edn` numbers the same as the defaults in `settings.clj`.